src/Node.hs

```diff
@@ -77,14 +77,16 @@ data ListenerAction packing m where
   -- | A listener that handles an incoming bi-directional conversation.
   ListenerActionConversation
     :: ( Packable packing snd, Unpackable packing rcv, Message rcv )
-    => (LL.NodeId -> ConversationActions snd rcv m -> m ())
```

Comment:
Well. Maybe usecase will arise. But as I understand now, it really brings lot more of responsibility providing interface to use `sendActions`. So we better use solution with workers and rollback code. Also I would prefer to strengthen this (not using sendActions from conversation) by logging error if use being detected (that is not hard to do with thread-local variables)